src/components/Form.js

```diff
@@ -1,25 +1,25 @@
 import React from 'react'
 import styled from 'styled-components/macro'
 import SaveBtn from './common/SaveBtn'
+import CategoryBtn from './common/CategorySelect'
```

Comment:
I would rename CategoryBtn to CategorySelect because the component is not a select element.